Formula/mariadb@10.2.rb

```diff
@@ -30,22 +29,33 @@ class MariadbAT102 < Formula
   depends_on "openssl@1.1"
   depends_on "pcre2"
 
+  uses_from_macos "bzip2"
+  uses_from_macos "ncurses"
+  uses_from_macos "zlib"
+
   on_macos do
     # Need patch to remove MYSQL_SOURCE_DIR from include path because it contains
     # file called VERSION
     # https://github.com/Homebrew/homebrew-core/pull/76887#issuecomment-840851149
-    # Reported upstream at https://jira.mariadb.org/browse/MDEV-7209 - this fix can be
-    # removed once that issue is closed and the fix has been merged into a stable release
+    # Originally reported upstream at https://jira.mariadb.org/browse/MDEV-7209,
+    # but only partially fixed.
     patch :DATA
   end
 
+  # This upstream commit was added for MariaDB 10.3+, but not 10.2. If it is not
+  # added in the next release, we should open an upstream PR to do so.
```

Comment:
This seems like a weird place to put this comment.

Reply:
Where should I move it to?  I'll do so in my syntax-only PR to add in the comment about the other fix.